open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconChooserPanel.java

```diff
@@ -149,6 +149,12 @@
     /** The optional selected icon URL. */
     private String mySelectedUrl;
 
+    /** The icon display width */
+	public int myTileWidth;
+	
+	 /** The list of icons being resized */
+	public List<IconRecord> ResizeRec = New.list();

```

Comment:
Field should start with my, or lowercase 'r' in this case.  Also tend to avoid abbreviations, such as Rec.  'myResizeRecord' would be a good name

Reply:
resolved